### DISCUSSION TITLE
FBX files with animation does not work correctly (with Simple Motion) in O3DE

### DISCUSSION THREAD

**Discussion by dnqbob** [o3de/o3de#6503] [Help Wanted]
Hi devs. 

When I tried to import some FBX files (with animation) to O3DE project and use them to learn O3DE's animation system, I found some problem (Actor + Simple Motion):

There is asset that works normally, like:
https://psionicgames.itch.io/spider-low-poly-animated

There are assets that work incorrectly, like:
https://quaternius.itch.io/lowpoly-animated-knight
https://kenney.nl/assets/animated-characters

Is there some different between the animation of humanoid model and other model in applying, or simply O3DE needs specifically processed models in order to get the correct response?

**Comment by dnqbob** [comment]
#6633 related

**Comment by watchmax3** [comment]
1- sure that model its OK and standard in model maker softwares.
2- check export fbx options in model maker software and edit setting in asset browser in o3de
3- actor is a component and simple motion another and if that not enough use animation editor tool
